lookout/style/format/features.py

```diff
@@ -163,6 +175,9 @@ def __init__(self, language: str, siblings_window: int = 5, parents_depth: int =
         language = language.lower()
         self.tokens = importlib.import_module("lookout.style.format.langs.%s.tokens" % language)
         self.roles = importlib.import_module("lookout.style.format.langs.%s.roles" % language)
+        self.interpreted_types = importlib.import_module(
+            "lookout.style.format.langs.%s.interpreted_types" % language) \
+            .INTERPRETED_TYPES
```

Comment:
We can remove the new line here, the whole thing is 91 chars long

Reply:
+ we need to catch ImportError here - there are some sane language drivers which do not do the shit